contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -3130,16 +3134,19 @@ void removeCtxStringFromQuery(PLtsql_expr* expr, ParserRuleContext *ctx, ParserR
 
 void extractQueryHintsFromOptionClause(TSqlParser::Option_clauseContext *octx)
 {
```

Comment:
nit: instead of touching existing code, adding the following lines would be an option.
```
if (!enable_hint_mapping)
    return; // do nothing
```